components/member-card/skills.js

```diff
@@ -0,0 +1,30 @@
+export const skills_data = {
+  topskills: [
+    {
+      tech: 'NodeJs',
+      background: '#3C873A'
+    },
+    {
+      tech: 'MongoDB',
+      background: '#3FA037'
+    },
+    {
+      tech: 'EmberJs',
+      background: '#CB4633'
+    }
+  ],
+  skills: [
+    {
+      tech: 'React',
+      background: '#35BBDE'
+    },
+    {
+      tech: 'Javascript',
+      background: '#D9C643'
+    },
+    {
+      tech: 'Firestore',
+      background: '#FFA611'
+    }
+  ]
+};
```

Comment:
can we please move this into a Mock Data folder? create a new folder in the root, `mock-data`, and move this file there.